extensions/variants.go

```diff
@@ -398,6 +424,13 @@ func (s *AggregateFunctionVariant) Match(argumentTypes []types.Type) (bool, erro
 func (s *AggregateFunctionVariant) MatchAt(typ types.Type, pos int) (bool, error) {
 	return matchArgumentAt(typ, pos, s.Nullability(), s.impl.Args, s.impl.Variadic)
 }
+func (s *AggregateFunctionVariant) MinArgumentCount() int {
+	return minArgumentCount(s.impl.Args, s.impl.Variadic)
+}
+
+func (s *AggregateFunctionVariant) MaxArgumentCount() int {
+	return maxArgumentCount(s.impl.Args, s.impl.Variadic)
```

Comment:
add a test?

Reply:
added tests for this, thanks for pointing this. I missed it